pylint/pyreverse/inspector.py

```diff
@@ -6,22 +6,30 @@
 
 Try to resolve definitions (namespace) dictionary, relationship...
 """
+from __future__ import annotations
+
 import collections
 import os
 import traceback
+from collections.abc import Callable, Iterator
+from typing import Any, Optional
 
 import astroid
 from astroid import nodes
 
 from pylint.pyreverse import utils
 
+_WrapperFuncT = Callable[[Callable[[str], nodes.Module], str], Optional[nodes.Module]]
```

Comment:
These need to be imported from `typing` since they are used at runtime.

Reply:
Fixed.